crowdsec-docs/docs/bouncers/cloudflare.mdx

```diff
@@ -152,6 +156,16 @@ Configuration file can be found at `/etc/crowdsec/bouncers/crowdsec-cloudflare-b
 crowdsec_lapi_url: http://localhost:8080/
 crowdsec_lapi_key: ${LAPI_KEY}
 crowdsec_update_frequency: 10s
+include_scenarios_containing: [] # ignore IPs banned for triggering scenarios not containing either of provided word
+exclude_scenarios_containing: # ignore IPs banned for triggering scenarios containing either of provided word
+    - ssh
+    - ftp
+    - smb
+only_include_decisions_from: # only include IPs banned due to decisions orginating from provided sources. eg value ["cscli", "crowdsec"]
```

Comment:
put the possible value in the paramter `references` instead of the example please